docs/maintenance-plan.md

```diff
@@ -0,0 +1,189 @@
+# Maintenance plan
+
+This document outlines every piece of the platform that need upgrading and
+updating.
+
+## Lagoon
+
+Update interval: 1-4 months
+Release URL: <https://github.com/uselagoon/lagoon/releases>
+Upgrade Docs URL:
+Expect downtime: Highly likely
+Time to do: Unknown at the time of writing (most likely a day the first time)
+Runbook: [Upgrading Lagoon](./runbooks/upgrading-lagoon.md)
+Notes:
+
+## AKS
+
+Update interval: monthly
+Release URL: <https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli>
+Upgrade Docs URL:
+  <https://learn.microsoft.com/en-us/azure/aks/upgrade-aks-cluster?tabs=azure-portal>
+Expect downtime: yes, there'll be a very short outage alike to releases
+Time to do: 2-6 hours
+Runbook: [Update AKS](./runbooks/upgrading-aks.md)
+Notes:
+
+## Support Workloads
+
+We have a number of support workloads. Which at the time of writing is all
+Helm Charts.
+Information on Helm upgrades can be found
+on the [Helm website](https://helm.sh/docs/helm/helm_upgrade/).
+There's a general runbook for upgrading the support workloads here:
+[Upgrade Support Workloads](./runbooks/upgrading-support-workloads.md).
+
+### Cert-manager
+
+Update interval: weekly-monthly
```

Comment:
Honestly we should be fine updating cert-manager annually or every 6 months, unless we learn of security updates.